crates/bdk/Cargo.toml

```diff
@@ -62,8 +62,10 @@ rustdoc-args = ["--cfg", "docsrs"]
 name = "mnemonic_to_descriptors"
 path = "examples/mnemonic_to_descriptors.rs"
 required-features = ["all-keys"]
+doc-scrape-examples = true
```

Comment:
I've never heard of this feature before. Interested to see how this works. Unfortunately our new examples are in their own crates so this won't work there.

Reply:
At least the `bdk` crate still has some code in it's own `examples`, and in the future it would be nice to put some simple examples in all the crates just to get the code added to the docs.

Reply:
Nevermind, I removed the `doc-scrape-examples` feature, can add back later if we want it.